compute-at-edge.witx

```diff
@@ -297,6 +297,12 @@
         (param $h $request_handle)
         (result $err (expected (error $fastly_status)))
     )
+
+    (@interface func (export "auto_decompress_response_set")
+        (param $h $request_handle)
+        (param $encodings $content_encodings)
```

Comment:
i think this is a new type! i suspect `typenames.witx` needs to be updated to include a `content_encodings` type